.github/workflows/node.js.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
```

Comment:
How about `master` branch? In case someone pushes to that branch in a  rush right before the hackathon.

Reply:
Thanks for that. I forgot about `master`!